engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -354,48 +464,80 @@ public static String removeComments(String s) {
         // look for (ignoring whitespace): optional "import" optional "static" everything_else optional ".*" optional
+        // "as" optional ".*" optional
         // semicolon
         // "everything_else" should be a valid java identifier of the form package.class[.class|.method|.field]. This
         // will be checked later
         Matcher matcher = Pattern
-                .compile("^\\s*(import\\s+)\\s*(?<static>static\\s+)?\\s*(?<body>.*?)(?<wildcard>\\.\\*)?[\\s;]*$")
+                .compile(
+                        "^\\s*(import\\s+)\\s*(?<static>static\\s+)?\\s*(?<body>.*?)(?<wildcard>\\.\\*)?(\\s+as\\s+(?<alias>.*?))?[\\s;]*$")
                 .matcher(importString);
         if (!matcher.matches()) {
-            return null;
+            return Optional.empty();
         }
         final boolean isStatic = matcher.group("static") != null;
         final boolean isWildcard = matcher.group("wildcard") != null;
         final String body = matcher.group("body");
+        @Nullable
+        final String alias = matcher.group("alias");
         if (body == null) {
-            return null;
+            return Optional.empty();
         }
+        final Optional<GroovyImport> result;
 
         boolean okToImport;
         if (isStatic) {
+            final String typeName;
+            @Nullable
+            final String memberName;
             if (isWildcard) {
+                if (alias != null) {
+                    return Optional.empty();
+                }
                 // import static package.class[.class].*
                 okToImport = classExists(body);
+                typeName = body;
+                memberName = null;
             } else {
                 // import static package.class.class
                 // import static package.class[.class].method
                 // import static package.class[.class].field
                 final int lastSeparator = body.lastIndexOf(".");
                 if (lastSeparator > 0) {
-                    final String prefix = body.substring(0, lastSeparator);
-                    final String suffix = body.substring(lastSeparator + 1);
-                    okToImport = functionExists(prefix, suffix) || fieldExists(prefix, suffix) || classExists(body);
+                    typeName = body.substring(0, lastSeparator);
+                    memberName = body.substring(lastSeparator + 1);
+                    okToImport = functionExists(typeName, memberName) || fieldExists(typeName, memberName)
+                            || classExists(body);
                 } else {
                     okToImport = classExists(body);
+                    typeName = body;
+                    memberName = null;
                 }
             }
+            if (okToImport) {
+                if (isWildcard) {
+                    result = Optional.of(imports -> imports.addStaticStars(body));
+                } else {
+                    if (alias != null) {
```

Comment:
I would collapse this `if` with `else` above.

Reply:
Or drop the else entirely and just return once? no opinions offered either way, so I'm going to apply that change and your suggestions...